test/extensions/clusters/redis/redis_cluster_integration_test.cc

```diff
@@ -36,7 +36,9 @@ const std::string& testConfig() {
         name: envoy.redis_proxy
         config:
           stat_prefix: redis_stats
-          cluster: cluster_0
+          prefix_routes:
```

Comment:
Q: are we losing coverage of the deprecated paths here and below with this change? Should these tests instead be wrapped or do we need additional tests?

Reply:
I can add unit tests for the old route logic.  Ok if I do it in a follow-up?  I'd like to land this sooner rather than later, just so I don't have to do more test fix up :-)

Reply:
Sure sounds good.